contributors.yaml

```
@@ -111,4 +111,8 @@ contributors:
     family-names: Ojha
     given-names: Hardik
     affiliation: "Indian Institute of Technology Roorkee"
+  -
+    family-names: Bhardwaj
+    given-names: Yash
+    affiliation: "BITS Pilani Goa Campus"
```

If it's okay with you, I would suggest spelling out the abbrevation. It looks like [Wikipedia uses this for the official name](https://en.wikipedia.org/wiki/Birla_Institute_of_Technology_and_Science,_Pilani_%E2%80%93_Goa_Campus).
```suggestion
    affiliation: "Birla Institute of Technology and Science, Pilani - Goa Campus"
```

---

Sure. I'll make the necessary changes